homeassistant/components/frontend/__init__.py

```diff
@@ -81,7 +94,26 @@
         vol.Optional(CONF_EXTRA_HTML_URL_ES5):
             vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(CONF_JS_VERSION, default=JS_DEFAULT_OPTION):
-            vol.In(JS_OPTIONS)
+            vol.In(JS_OPTIONS),
+        vol.Optional(CONF_MANIFEST_JSON, default={
+            CONF_MANIFEST_JSON_DISPLAY: CONF_MANIFEST_JSON_DISPLAY_DEFAULT,
+            CONF_MANIFEST_JSON_SHORT_NAME:
+                CONF_MANIFEST_JSON_SHORT_NAME_DEFAULT,
+            CONF_MANIFEST_JSON_START_URL: CONF_MANIFEST_JSON_START_URL_DEFAULT
+        }): vol.Schema({
+            cv.string: cv.string,
+            vol.Optional(
+                CONF_MANIFEST_JSON_DISPLAY,
```

Comment:
Why would we want to make display configurable?

Reply:
Actually this is the main reason I proposed this modifications - started from a personal need. In standalone mode when you open the app in Chrome for Android it always opens in a new Chrome Tab. Of course being lazy I do not close the tab and I end up with multiple Home Assistant tabs opened in Chrome which slows the browser considerable. On the other hand switching the display to 'fullscreen' mode opens the app in a separate standalone webview windows - without the browser controls - but I never use those for HA anyway and I think most users don't.

Reply:
I don't think that this is a good use case. You're proposing to add a configuration option to work around an issue. Now each user will need to fix this for themselves.

https://developers.google.com/web/fundamentals/web-app-manifest/#display

What you're experiencing is not correct. Do you access your Home Assistant instance over https?

Reply:
I do access HA over SSL but I do not think this has anything to do with Chrome behavior - not sure though I will have to test this. Anyway I tested on 2 mobile devices with 2 different Android versions unfortunately same Chrome version and revealed the same behavior. I do not think this is an issue for all users - some users might want the browser controls when opening HA but some  won't.

Reply:
Could you
- enable developer settings on Android and enable USB debugging
- open Chrome on an Android phone
- connect the phone to your computer
- open Chrome on your desktop and navigate to chrome://inspect/#devices
- inspect the Chrome tab on your phone that has HA open
- Click on the application tab
- Click on manifest
- Click on "Add to homescreen"
- See if an error is printend to your console why a site cannot be added to your home screen.

Reply:
Because I still think that we should fix this instead of allowing you to bypass the problem with a config option and leaving other users in a broken state.

Reply:
Already validated manifest.json with Chrome Dev tools as well as a web based service (which I did not trust as much as Chrome itself) - no errors found.

Actually I have been investigating this most of last night and it seems to be a Chrome issues related to the fact that I am accessing HA with https://<host>:<port> and the port part seems to be the issue, Found this on a StackOverflow post so not 100% sure this is the issue but I am trying to set up an apache proxy now to test this. 
Either way you are right - we can close this PR

Reply:
I can confirm this is in fact the issue. When HA is accessible on a URL in the format https://<host>:<port>/ clicking on the link created with Add to Homescreen opens a new Chrome tab everyy time and the URL is shownin the opened tab - basically the standalone setting of display in manifest.json is ignored. When accessing HA on a URL like https://<host> everything works fine. So we migth want to include this information in the docs.